salt/journal/init.sls

```diff
@@ -116,7 +124,7 @@ composer-install:
             - COMPOSER_DISCARD_CHANGES: 'true'
         - require:
             - file: config-file
-            - cmd: var-directory
+            - journal-cache-clean
```

Comment:
could `SYMFONY_ENV` be removed from this state or mark as deprecated?

Reply:
No, the Composer scripts still run. (The cache clean will do nothing and the other script doesn't need an env, but better to safe.)